src/applications/mhv/medications/components/MedicationsList/MedicationsListCard.jsx

```diff
@@ -45,7 +45,9 @@ const MedicationsListCard = props => {
             id="status"
             className="vads-u-margin-top--1p5 vads-u-font-weight--bold"
           >
-            {rx.dispStatus}
+            {rx.dispStatus !== 'Active: Refill in Process'
```

Comment:
Should we be using a constant here?

Reply:
I agree that would be a good idea. I would want to replace other places where this logic is used as well if so.